src/coreclr/jit/codegenxarch.cpp

```diff
@@ -10452,8 +10454,10 @@ void CodeGen::genFnEpilog(BasicBlock* block)
     }
 #endif
 
+    genClearAvxStateInEpilog();
+
     // Restore float registers that were saved to stack before SP is modified.
-    genRestoreCalleeSavedFltRegs(compiler->compLclFrameSize);
+    genRestoreCalleeSavedFltRegs();
```

Comment:
Similar question here around the epilog and just calling `genClearAvxStateInEpilog` as part of `genRestoreCalleeSavedFltRegs`